README.rst

```diff
@@ -91,9 +90,8 @@ use, please see the following resources:
 
 Below is a list of tested compiler/OS combinations:
 
-- GCC 7.5.0 (Ubuntu 20.04)
 - GCC 11.3.0 (Ubuntu 22.04)
```

Comment:
I should have seen this earlier, the GCC used according to https://github.com/CppMicroServices/CppMicroServices/actions/runs/13314697423/job/37185735342?pr=1063 is `11.4.0`